src/chrome/content/rules/Tencent_CN.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+	Might be blocked by The Great Fire Wall (GFW) in China, Check error: Connection timed out ( after 20000 milliseconds ).
+	Test in Tor broswer 6.0.4, Ok by http, however, unable to connect to https.
+	Users could turn on this rule to test by youself.
```

Comment:
I have did the tests again:

![sp160914_193235](https://cloud.githubusercontent.com/assets/2649544/18510727/345cb126-7ab2-11e6-85cf-fa7168e72dc5.png)

shp.qpic.cn maybe ok.